upgrade_tests/upgrade_manifest.py

```diff
@@ -160,38 +160,15 @@ def clone_with_local_env_version(self):
 #   4) If a new sstable format is present in version B, writes will occur in that format after upgrade. Running sstableupgrade on version B will proactively convert version A sstables to version B.
 # TODO define new upgrade scenarios whenever Cassandra is branched
 MANIFEST = {
-    indev_2_1_x: [indev_2_2_x, current_2_2_x, indev_3_0_x, current_3_0_x, indev_3_11_x, current_3_11_x],
-    current_2_1_x: [indev_2_1_x, indev_2_2_x, current_2_2_x, indev_3_0_x, current_3_0_x, indev_3_11_x, current_3_11_x],
-
-    indev_2_2_x: [indev_3_0_x, current_3_0_x, indev_3_11_x, current_3_11_x],
-    current_2_2_x: [indev_2_2_x, indev_3_0_x, current_3_0_x, indev_3_11_x, current_3_11_x],
-
-    indev_3_0_x: [indev_3_11_x, current_3_11_x, indev_trunk],
-    current_3_0_x: [indev_3_0_x, indev_3_11_x, current_3_11_x, indev_trunk],
-
+    current_2_1_x: [current_2_2_x, indev_3_0_x, indev_3_11_x],
+    current_2_2_x: [indev_3_0_x, indev_3_11_x],
+    current_3_0_x: [indev_3_0_x, indev_3_11_x, indev_trunk],
```

Comment:
do we need `indev_3_0_x` here?

Reply:
it depends, is 30 still supported? if so - yes, if not, then no :)

Reply:
gotcha. then there should be a `current_2_2_x -> indev_2_2_x` too, since that is currently supported (and will be accepting critical bug fixes for ~one more year)

Reply:
i didn't know that; i'll bring it back then